configs/16.0/release_notes/release_notes-inst.html

```diff
@@ -151,8 +151,37 @@ <h3>Using DNF</h3>
 				<p>Answer <span class="code">y</span> when DNF prompts for confirmation.</p>
 endif_RHEL8 -->
 
+<!-- if_RHEL9
+			<p>The gpg public key(s) <span class="code">gpg-pubkey-615d762f-62f504a1</span> will be provided in the repository where these release notes were found.  The pubkey(s) can be used to verify the authenticity of both the Advance Toolchain install packages and the repository contents. After downloading the gpg-pubkey(s) they can be imported as follows:</p>
+			<p class="code">rpm --import gpg-pubkey-615d762f-62f504a1</p>
```

Comment:
other similar sections use "gpg-pubkey-xxxxxxxx-xxxxxxxx" (literally). any reason for the difference here?